example_sns_init.yaml

```diff
@@ -0,0 +1,117 @@
+# You should make a copy of this file, name it sns_init.yaml, and edit it to
```

Comment:
Is it possible to configure the exact name when running `sns propose` or does the file have to be called sns_init.yaml and present in the current working directory?

Reply:
> Currently, propose_sns.sh does not have this feature. Instead, it assumes sns_init.yaml.

I would add the command-line argument to make it explicit how it can be used (analogously to how deploy_sns.sh handles the yaml file).

Reply:
Done.